test/WebJobs.Script.Tests/Workers/Rpc/GrpcMessageConversionExtensionsTests.cs

```diff
@@ -485,6 +485,23 @@ public async Task ToRpc_Collection_String_Without_Capabilities_Value()
             Assert.Equal(typedData.Json, returned_typedata.Json);
         }
 
+        [Fact]
+        public async Task ToRpc_Collection_String_IncludeEmptyEntries_for_TypedDataCollection()
```

Comment:
nit - name seems a little disjointed? Is "Rpc collection" a single concept? (e.g. would "ToRpcCollection_String_IncludeEmptyEntriesForTypedDataCollection" make sense?)

Reply:
ToRpc is the method name and other tests are using this format. So I will keep it as it is for consistency.